### DISCUSSION TITLE
Building Workflows

### DISCUSSION THREAD

**Discussion by elkhawajah** [elsa-workflows/elsa-core#63] [Q&A]
Hi guys - 

Anyone interested in a remote work here ? I need someone with expertise in Elsa workflow to build  workflows for a performance  management  app currently  under development.

You will join a team, and communication will be over slack.

You can email me to ahmad@tasqat.com

**Comment by sfmskywalker** [comment]
I'd be interested :)

**Comment by elkhawajah** [comment]
@sfmskywalker Lets chat over email ?
